lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -140,6 +140,16 @@ def answer_labels
             translate(answer_string)
           rescue TypeError
             "unknown answer label"
+          rescue NoMethodError
```

Comment:
you can do `rescue NoMethodError => e` and then just re-raise the e for the original error report.